### ISSUE TITLE
Superscript number glyphs are centered in the display box, obsuring the fact that they're superscripts

### ISSUE THREAD

**Issue by JoeUX** [EdiWang/Character-Map-UWP#22]
Check out Palatino Linotype, Calibri, or Open Sans (if you have it installed). Look at character U+2074 and the few after it. These numerals are centered in the box, but they're superscripts. They should be pushed to the top to make clear what they are. The way they're currently presented, it's not clear if they're just normal numbers, subscripts, or superscripts.

Notably, the subscript numbers are not centered – they're at the bottom of the box where they belong. (See U+2084.)

**Comment by verdy-p** [comment]
Yes, the problem is that CMUWP just considers the glyph bounding box, without taking into account the font metrics for including at least the basic character cell (including all baselines, that should be ideally be renderable in the background using thin grayed lines, and the initial and final advance position on the orthogonal axis)

Also there's still no way to specify the vertical rendering style (where the baseline becomes vertical: not all glyphs are simply rotated), or mirrored styles (for mirrorable characters in BiDi), no way to view a list of supported variants of the character (including in contextual positions, or in other implemented OpenType features), we can just select a very restricted subset of "axis" (for varaible fonts).

But OpenType features are still MUCH MORE important than those variable axis (notably, many scripts cannot be rendered at all without these features, some of them being mandatory, and several others being recommanded).

Is there work to allow to browse ALL glyphs in a font, and not just that are in its default Unicode cmap? I mean: a complete list of OpenType features (per locale, notably for CJK local variants, and per feature type e.g. "isol", "init", "medl", "finl"). For now we can only see the isolated form, excluding all other featured ligatures, or additional typographic features (such as kerning pairs, also needed for correct rendering of glyphs).

Font-level info also do not display all information: basic metrics, licences and usage restrictions, design locales and supported locales; we just have a few PANOSE fields.

"Developer tools" only work at single Unicode character level, not for all glyphs or for a whole string in the editor, or a selected part of it. They do not show glyphs that are in fact a composition from other glyphs (made with SUBSTITUTION and POSITION tables). We can't see the effective names recommanded in font development formats, (including with their recommanded standard names from PostScript or "uniXXXX" names possibly with "."-suffixes).
